doc/user/methods/pca.md

```diff
@@ -0,0 +1,326 @@
+## `PCA`
+
+The `PCA` class implements principal components analysis (PCA), a standard
+machine learning data preparation technique.  PCA can be used to reduce the
+number of dimensions in a dataset, or to preserve a certain percentage of the
+variance of a dataset.
+
+By default, `PCA` uses the full exact singular value decomposition (SVD), but
+supports the use of other more efficient decompositions, including approximate
+singular value decompositions.
+
+#### Simple usage example:
+
+```c++
+// Use PCA to reduce the number of dimensions to 5 on uniform random data.
+
+// This dataset is uniform random in 10 dimensions.
+// Replace with a data::Load() call or similar for a real application.
+arma::mat dataset(10, 1000, arma::fill::randu); // 1000 points.
+
+mlpack::PCA pca;       // Step 1: create PCA object.
+pca.Apply(dataset, 5); // Step 2: reduce data dimension to 5.
+
+// Print some information about the modified dataset.
+std::cout << "The transformed data matrix has size " << dataset.n_rows /* 5 */
+    << " x " << dataset.n_cols << "." << std::endl;
+```
+<p style="text-align: center; font-size: 85%"><a href="#simple-examples">More examples...</a></p>
+
+#### Quick links:
+
+ * [Constructors](#constructors): create `PCA` objects.
+ * [`Apply()`](#applying-transformations): apply PCA transformation to data.
+ * [Examples](#simple-examples) of simple usage and links to detailed example
+   projects.
+ * [Template parameters](#advanced-functionality-different-decomposition-strategies)
+   for using different decomposition strategies.
+
+#### See also:
+
+<!-- TODO: add link -->
```

Comment:
I guess something for later?

Reply:
Right, I can't add the link to RADICAL until the documentation page for that method is done.